Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -200,7 +200,7 @@ def GetPlatformDscAndConfig(self) -> tuple:
 
         The tuple should be (<workspace relative path to dsc file>, <input dictionary of dsc key value pairs>)
         '''
-        return ("QemuQ35Pkg/QemuQ35Pkg.dsc", {})
+        return ("QemuQ35Pkg/QemuQ35PkgX64.dsc", {})

```

Comment:
Should this also return QemuQ35PkgIA32X64.dsc?

Reply:
OvmfPkg does [this](https://github.com/tianocore/edk2/blob/1c74842bd07a48070d1cf3458f0b6b377e3ec8b9/OvmfPkg/PlatformCI/PlatformBuildLib.py#L106-L113):

```python
    def GetPlatformDscAndConfig(self) -> tuple:
        ''' If a platform desires to provide its DSC then Policy 4 will evaluate if
        any of the changes will be built in the dsc.

        The tuple should be (<workspace relative path to dsc file>, <input dictionary of dsc key value pairs>)
        '''
        dsc = CommonPlatform.GetDscName(",".join(self.ActualArchitectures))
        return (f"OvmfPkg/{dsc}", {})
```

Reply:
Good call. Thanks. This is updated.